docs/UserGuide.adoc

```diff
@@ -272,55 +287,43 @@ Example:
 +
 `deleteIngredient 3`
 
-Delete chicken from ingredient inventory
+Delete chicken from inventory
 
 ==== List Recipes: `listRecipes`
 
-View recipes of dishes served at the restaurant.
+List all recipes.
 
 Format: `listRecipes`
 
-****
-• For each recipe, the list of ingredients and its corresponding amount in units required for the standard serving will be displayed.
-Each recipe will be indexed.
-****
 
 ==== Add Recipe: `addRecipe`
 
 Add recipe of dish served at the restaurant.
 
-Format: `addRecipe r/RECIPE_NAME n/NUMBER_OF_INGREDIENTS 1/INGREDIENT_1&AMOUNT_IN_UNITS 2/INGREDIENT_2&AMOUNT_IN_UNITS ...`
+Format: `addRecipe r/RECIPE_NAME iq/INGREDIENT_INDEX&INGREDIENT_QUANTITY__IN_RECIPE ...`
 
 ****
-• For all ingredient inputs, if ingredient is new, message appears as a prompt for user to `addIngredient` first, before adding recipe.
+• The index refers to the index number shown in the most recent listing of ingredients in the ingredients panel.
 
-• Each ingredient input should have the format `1/INGREDIENT_1&AMOUNT_IN_UNITS` where ingredient and amount in standardised units are separated by `&`.
+• There should be at least 1 ingredient `iq/INGREDIENT_INDEX&INGREDIENT_QUANTITY_IN_RECIPE` input
 
-• `AMOUNT_IN_UNITS` is allowed to take on a decimal and is in the units specified in listIngredients.
-
-• Number of ingredient inputs should correspond to `n/NUMBER_OF_INGREDIENTS`. If it does not, a message will appear to say incorrect number of inputs.
+• Each ingredient input should have the format `iq/INGREDIENT_INDEX&NGREDIENT_QUANTITY_IN_RECIPE`
+where ingredient index and quantity of ingredient in recipe is separated by &.
+Both the ingredient index and quantity must be an whole number.
```

Comment:
is 0 allowed for quantity?

Reply:
No, the ingredient must be non-zero because the recipe must contain at least 1 unit of that ingredient. I will indicate this in the next draft of user guide, and in my implementation